string agent_name
int x
int y
int theta
---
vector<Point> path
